app.js

```diff
@@ -71,19 +110,35 @@ var postingStrategies = {
   file: postFile
 };
 
-bot.onPinAdded(function (message, channelData) {
+var tweacji = (env.REACJI_TO_TRIGGER_TWEET || 'pushpin').split(',');
+var tweacjiFormatted = tweacji.map(function (ji) { return `:${ji}:`; });
+var tweacjiList = tweacjiFormatted.pop();
+if (tweacjiFormatted.length > 0) {
+  tweacjiList = tweacjiFormatted.join(', ') + ' or ' + tweacjiList;
```

Comment:
Is this all to be able to produce a nicely formatted DM?

Reply:
Yeah. I guess I could use some "humanize" library, but when it's four lines of code...